Intrepid.podspec

```diff
@@ -36,7 +36,7 @@ Pod::Spec.new do |s|
 
   s.subspec "Rx" do |rx|
     rx.source_files = "SwiftWisdom/Rx/**/**/*.swift"
-    rx.dependency 'RxSwift', '~> 4.0'
-    rx.dependency 'RxCocoa', '~> 4.0'
+    rx.dependency 'RxSwift', '~> 4.2'
```

Comment:
Was I wrong about this? I can't find anything about swift 4.2 support on the RxSwift repo. Maybe `RxSwift 4.0` supports swift 4.2? 😕

Reply:
It looks like `4.3.1` might, but I'm not sure. I can't find very much concrete documentation either.

Reply:
@jpmendel Found that we actually need this to be 4.3.1 according to their release notes.